app/pandas.ipynb

```diff
@@ -549,14 +914,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here\n"
-   ],
+   "source": "# write your code here\n",
```

Comment:
The code cell for the Pandas and Numpy solution to calculate the moving average is currently empty. You need to implement this part to complete Task 6. Consider using the `rolling` method from Pandas to calculate the moving average efficiently.